### PR TITLE
feat(voltage-init-parameters): add parameters related to study and voltage-init

### PR DESCRIPTION
**Pull request by TheMaskedTurtle**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/repository/StudyEntity.java [hunk 1]
```diff
@@ -172,6 +173,10 @@ public class StudyEntity extends AbstractManuallyAssignedIdentifierEntity<UUID>
     @Builder.Default
     private StudyIndexationStatus indexationStatus = StudyIndexationStatus.NOT_INDEXED;
 
+    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "study_voltage_init_parameters_id")
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/study/server/repository/StudyEntity.java:177]: Set explicit and readable name for foreign key constraint

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/study/server/repository/StudyEntity.java (hunk 1 above)
Comment (TheMaskedTurtle) [reply src/main/java/org/gridsuite/study/server/repository/StudyEntity.java:177]: I thought that was quite explicit already, it's parameters related to both study and voltage init parameters. Do you have any other idea ?

Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/study/server/repository/StudyEntity.java:177]: It's for the foreign key name
See securityAnalysisParameters in the same entity just above with JPA annotation : 
foreignKey = @ForeignKey(
                    name = "securityAnalysisParameters_id_fk"
            ))

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/study/server/repository/StudyEntity.java (hunk 1 above)
Comment (TheMaskedTurtle) [reply src/main/java/org/gridsuite/study/server/repository/StudyEntity.java]: ok fixed !

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**